### DISCUSSION TITLE
Logo Ideas

### DISCUSSION THREAD

**Discussion by Ancyker** [sanni/cartreader#673] [Ideas & Suggestions]
Was messing around with logo ideas...

![oscr-oshw-rect_logo-white](https://user-images.githubusercontent.com/7105333/210455115-b9ab8459-eae8-4b30-89bd-8c51c276d39d.png)
![oscr-oshw-rect_logo](https://user-images.githubusercontent.com/7105333/210455124-67983ecb-1822-4ac5-8e84-04a000c9ff5d.png)
<img width="724" alt="Screenshot_1001" src="https://user-images.githubusercontent.com/7105333/210455133-f38470f0-d176-464b-8082-6e90acb44ee1.png">
<img width="628" alt="Screenshot_1002" src="https://user-images.githubusercontent.com/7105333/210455141-4ee9a2b1-9a78-4e44-810e-fc6bee08d2f2.png">

**Comment by sanni** [comment]
Very nice. 👍 

One idea I had was something like this:

![fasfasaffaa](https://user-images.githubusercontent.com/19169494/213197258-73b89b55-44c4-4794-8c1e-4ee73d2dfe3e.jpg)

Where each outline is a recognizable cartridge shape.

**Reply by Ancyker** [reply]
I was playing around with cartridges but didn't like how they looked when turned into a monochrome logo on a PCB.

Here's one:
![](https://cdn.discordapp.com/attachments/1038265106179575911/1060060918475657266/image.png)

**Comment by Cephiros** [comment]
I think you could get the best of both worlds, use the open source hardware logo on top, and sanni's cartridge letters below. Maybe, as shown, NES over the O, and Gameboy over the S, though that about does it for tall recognizable cartridge outlines, you may be able to turn the outline on the side and put the CD in an n64, snes or genesis carts, or put two letters in either horizontally.

additionally you can use less recognized carts that are still dumped like the pcengine, game gear, or virtual boy.
